Jenkinsfile_CNP

```diff
@@ -87,6 +87,17 @@ withPipeline(type, product, component) {
       builder.gradle('functional')
     } finally {
       junit '**/test-results/**/*.xml'
+      junit 'build/test-results/functional/**/*.xml'
+      archiveArtifacts 'build/reports/tests/functional/index.html'
+
+      publishHTML target: [
+        allowMissing         : true,
+        alwaysLinkToLastBuild: true,
+        keepAll              : true,
+        reportDir            : "build/reports/tests/functional",
+        reportFiles          : "index.html",
+        reportName           : "JRD Functional Test Report"
```

Comment:
I think this should be LRD instead of JRD

Reply:
Yes correct :)